ferrocene/tools/self-test/src/binaries.rs

```diff
@@ -83,9 +87,9 @@ fn parse_version_output(output: &str) -> Option<VersionOutput<'_>> {
 
         let Some((key, value)) = line.split_once(": ") else { continue };
         match key {
-            "host" => host = Some(value),
-            "commit-hash" => commit_hash = Some(value),
-            "release" => release = Some(value),
+            "host" => host = Some(value.to_string()),
+            "commit-hash" => commit_hash = Some(value.to_string()),
+            "release" => release = Some(value.to_string()),
```

Comment:
nit: `.into()` should work here, I think

Reply:
I used `into` whereever possible